src/main/java/uk/gov/companieshouse/lookup/exception/GlobalExceptionHandler.java

```diff
@@ -5,29 +5,21 @@
 import org.springframework.web.bind.annotation.ControllerAdvice;
 import org.springframework.web.bind.annotation.ExceptionHandler;
 import org.springframework.web.bind.annotation.ResponseStatus;
-import java.util.logging.Logger;
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.logging.LoggerFactory;
+import uk.gov.companieshouse.lookup.Application;
 
 @ControllerAdvice
 public class GlobalExceptionHandler {
 
-    //private static final Logger LOG = LoggerFactory.getLogger(Application.APPLICATION_NAME_SPACE);
-    private final static Logger LOG = Logger.getLogger(GlobalExceptionHandler.class.getName());
+    private static final Logger LOG = LoggerFactory.getLogger(Application.APPLICATION_NAME_SPACE);
 
-    @ExceptionHandler(value = { RuntimeException.class })
-    @ResponseStatus(value = HttpStatus.INTERNAL_SERVER_ERROR)
-    public String handleRuntimeException(HttpServletRequest request, Exception ex) {
-
-        LOG.severe(ex.getMessage());
-//
-        return "error";
-    }
 
-    @ExceptionHandler(value = { ServiceException.class })
+    @ExceptionHandler({RuntimeException.class, ServiceException.class})
```

Comment:
ServiceException is a checked exception so shouldn't need to be included here

Reply:
It's a wrapper for the other exceptions which I've handled as much as I can, this is the default behaviour for unrecoverable exceptions.

Reply:
What I mean to say is `ServiceException` is checked so will be caught explicitly in the code; only `RuntimeException`'s need to be caught here as they won't be caught explicitly within a try / catch block

Reply:
I understand ServiceException is a checked exception but that doesn't stop you adding it to the controller method signature when no specific recoverable scenario exists. This pushes the responsibility for this error handling further up the chain and onto the boiler plate controller advice code. The alternative is to add log and error scenarios to both controllers which doesn't seem ideal.